BlockBlobReader/src/consumer.js

```diff
@@ -116,12 +118,45 @@ function csvHandler(msgtext, headers) {
     return messageArray;
 }
 
+function nsgLogsHandler(jsonArray) {
```

Comment:
This function needs to be modified for additional fields 
{ flow_state = $splitted_tuple[8];}}
{{ num_packets_sent_src_to_dest = $splitted_tuple[9];}}
{{ bytes_sent_src_to_dest = $splitted_tuple[10];}}
{{ num_packets_sent_dest_to_src = $splitted_tuple[11];}}
{{ bytes_sent_dest_to_src = $splitted_tuple[12]}}

Also, can you add version field to it ?